src/js/components/Tag/propTypes.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import {
+  backgroundPropType,
+  genericProps,
+} from '../../utils/general-prop-types';
+
+let PropType = {};
+if (process.env.NODE_ENV !== 'production') {
+  PropType = {
+    ...genericProps,
+    as: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
+    background: backgroundPropType,
```

Comment:
missing border shown in typescript, but, I'm wondering if we should omit this for the initial version?